### PR TITLE
Give execute procedure permission.

### PR DESCRIPTION
**Pull request by ishan1729**
Needs internal R&D review.

### CODE REVIEW

File: content/developerportal/deploy/setting-up-the-database-user.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ With the User Mapping properties you can grant the user access to the different
...
+The recommended database role is `db_owner`. If you choose not to assign the `db_owner` role to the user, the following roles should be assigned. The roles `db_datareader`, `db_datawriter` and execute procedure permission (`GRANT EXECUTE ON OBJECT::usp_nextsequencevalue TO <YourDatabaseUser>`) are the minimum permission the user should get so he can read and write in the contents of all the tables. Additionally, to start the application from a new `.mda` file the `view definition` permission is required, even if no database synchronization is necessary. During the database upgrade, altering the tables and stored functions is required. During that action, it is usually sufficient to have the `db_ddladmin` role, but when the stored functions needs to be added, the `db_owner` role is required (for more information, see [SQL Server Troubleshooter](troubleshooting-sql-server)).
```
Comment (passalaqua) [inline content/developerportal/deploy/setting-up-the-database-user.md:38]: I would prefer to split the _what_ with the _how_ to give permissions.

Maybe just "(...) execute procedure permission on `usp_nextsequencevalue` are the minimum (...)" and then add an example set of commands afterwards?

Comment (passalaqua) [reply content/developerportal/deploy/setting-up-the-database-user.md:38]: Doc scribbles, feel free to close this if you believe my suggestion does not add clarity. :D

Comment (ishan1729) [reply content/developerportal/deploy/setting-up-the-database-user.md]: Good idea. Made some changes.

Comment (passalaqua) [reply content/developerportal/deploy/setting-up-the-database-user.md]: I like the new text. Approved as far as I can say.

**Review by passalaqua — APPROVED** [review]

**Review by MarkvanMents — APPROVED** [review]
Looks OK to me. Can it be merged immediately?

**Comment by ishan1729** [comment]
> Looks OK to me. Can it be merged immediately?

Yes, we think so.

**Pull request merged**